src/main/java/io/cdap/plugin/db/batch/action/vertica/export/VerticaBulkExportAction.java

```diff
@@ -79,9 +79,10 @@ public void run(ActionContext context) throws Exception {
       int columnCount = metadata.getColumnCount();
 
       Configuration conf = new Configuration();
-      FileSystem fileSystem = FileSystem.get(conf);
-      Path exportFile = new Path(config.getPath());
+      String path = config.getPath();
+      Path exportFile = new Path(path);
       Path exportDir = exportFile.getParent();
+      FileSystem fileSystem = exportFile.getFileSystem(conf);
```

Comment:
what's the difference between this and FileSystem.get()? Seems like a really easy mistake to make, wondering if there is something we can do at the app/platform level.

Reply:
FileSystem.get(conf) returns `fs.defaultFS` where as `exportFile.getFileSystem` call returns the file system from URI.

Reply:
ah I see.